### DISCUSSION TITLE
failed to update cluster operator from 0.22.1 to 0.23.0

### DISCUSSION THREAD

**Discussion by islet5** [strimzi/strimzi-kafka-operator#7589] [Q&A] [answered]
I updated operator from 0.20.0 to 0.21.1, then update to 0.22.1, and seems everything is fine
![image](https://user-images.githubusercontent.com/9824473/200490693-8e18c6dd-2d66-4445-a069-912369d76e10.png)

kafka cluster is also updated
![image](https://user-images.githubusercontent.com/9824473/200490983-cfc0f30b-1b44-4dee-894f-f04b3ad723ab.png)

when i was trying to update kafka apiVersion, found it's already v1beat2
![image](https://user-images.githubusercontent.com/9824473/200491130-3cc642c0-12ee-4f2b-a19c-0ed671fe0821.png)

Then I try to update operator from 0.21.1 to 0.23.0, i got some errors
![image](https://user-images.githubusercontent.com/9824473/200491950-536dc0c7-7549-417a-92fd-a98f59ba7c7d.png)

**Comment by scholzj** [comment, marked as answer]
It looks like you didn't do the CRD conversion. Please follow the docs to convert the CRDs before installing 0.23.0.  Check the docs for the details. Also, you cannot upgrade from 0.21 to 0.23. You have to first go to 0.22 and then upgrade to 0.23 or newer after you do the CRD conversion.

**Reply by islet5** [reply]
sorry, that was a mistake, i was going to say "Then I try to update operator from 0.22.1 to 0.23.0, i got some errors". Thx, i will try to convert first.
